Sources/LocalReceiptParsing/PurchasesReceiptParser.swift

```diff
@@ -88,7 +102,7 @@ private extension ReceiptParser {
     static func ensureRunningOutsideOfMainThread() {
         // Only checking on integration tests.
         // Unit tests might run on the main thread when testing this class directly.
-        if ProcessInfo.isRunningIntegrationTests {
+        if ProcessInfo.processInfo.environment["RCRunningIntegrationTests"] == "1" {
```

Comment:
Don't know if there is a way in iOS to know if it's running tests generically so it's done automatically even for other people using this as a library... This should be ok for now though.

Reply:
There is, but we decided to use a separate variable: https://github.com/RevenueCat/purchases-ios/pull/2100#discussion_r1042638671